formats/MrssFormat.php

```diff
@@ -84,17 +84,17 @@ public function stringify(){
 			$feedImage->appendChild($iconLink);
 		}
 
-		$linkAlternate = $document->createElementNS(self::ATOM_NS, 'atom:link');
+		$linkAlternate = $document->createElementNS(self::ATOM_NS, 'link');
+		$channel->appendChild($linkAlternate);
 		$linkAlternate->setAttribute('rel', 'alternate');
 		$linkAlternate->setAttribute('type', 'text/html');
 		$linkAlternate->setAttribute('href', $uri);
-		$channel->appendChild($linkAlternate);
 
-		$linkSelf = $document->createElementNS(self::ATOM_NS, 'atom:link');
+		$linkSelf = $document->createElementNS(self::ATOM_NS, 'link');
```

Comment:
Was this change from `atom:link` to `link` intentional?

Reply:
Yes, thanks to PHP’s reconciliation algorithm, the namespace prefix is redundant. I wanted to add it to the commit message but then I forgot.